package io.pelt.hlam.auth.exceptions;

public class UserNotFoundException extends Exception {
}
